build/commands/lib/test.js

```diff
@@ -63,12 +64,14 @@ const test = (passthroughArgs, suite, buildConfig = config.defaultBuildConfig, o
 
   braveArgs = braveArgs.concat(passthroughArgs)
 
+  let buildTarget = suite
+  if (suite === 'brave_unit_tests' || suite === 'brave_browser_tests')
+    buildTarget = "brave/test:" + suite
+
+  config.buildTarget = buildTarget
+
   // Build the tests
-  if (suite === 'brave_unit_tests' || suite === 'brave_browser_tests') {
-    util.run('ninja', ['-C', config.outputDir, "brave/test:" + suite], config.defaultOptions)
-  } else {
-    util.run('ninja', ['-C', config.outputDir, suite], config.defaultOptions)
-  }
+  build(buildConfig, options)
```

Comment:
this should just be `util.buildTarget()`

Reply:
Then you need to manually update `options` with `config`, otherwise some config is not passed through. `build` does it for us.

Reply:
that is already done here https://github.com/brave/brave-core/pull/8457/files#diff-ae08efb1b3db917a9b74c8758cdfc636bc34464ebf56fcc454d4acc619e3d05bR26. This is not correct and it does a bunch of other things that we don't want or need.

Reply:
Ok I know what was going on. At the time I made the change `config.js` hasn't been updated and `use_goma` was not in `update`. It is fixed so yes, this shall work now.